girder/utility/server.py

```diff
@@ -238,8 +235,7 @@ def setup(test=False, plugins=None, curConfig=None):
     cherrypy.tree.mount(None, routeTable[constants.GIRDER_STATIC_ROUTE_ID],
                         {'/':
                          {'tools.staticdir.on': True,
-                          'tools.staticdir.dir': os.path.join(constants.STATIC_ROOT_DIR,
-                                                              'clients/web/static'),
+                          'tools.staticdir.dir': constants.STATIC_ROOT_DIR,
```

Comment:
This constant is now the *actual* static root so we don't have to append `clients/web/static` everywhere.  Because this is configurable, I think we will eventually want to move it to `config`.